TravelAgency-spring-mvc/src/main/java/cz/muni/fi/travelAgency/controllers/CustomerController.java

```diff
@@ -72,21 +79,25 @@ public String getEdit(@PathVariable long id, Model model) {
         return "customer/edit";
     }
 
-    @RequestMapping(value = "/editCustomer", method = RequestMethod.POST)
+    @RequestMapping(value = "editCustomer", method = RequestMethod.POST)
     public String submitEdit(@Valid @ModelAttribute("customer") CustomerCreateDTO createDTO,
                              BindingResult bindingResult, Model model, RedirectAttributes redirectAttributes,
                              UriComponentsBuilder uriBuilder) {
-        if (!resultIsValid(bindingResult, model)) {
+//        if (!resultIsValid(bindingResult, model)) {
```

Comment:
If this code is not needed, remove it. (I think it's not as we do not have any additional validation for customer other than the annotations)